src/test/kotlin/dev/stalla/builder/validating/ValidatingPodcastindexPersonBuilderTest.kt

```diff
@@ -0,0 +1,93 @@
+package dev.stalla.builder.validating
+
+import assertk.all
+import assertk.assertAll
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isFalse
+import assertk.assertions.isNotNull
+import assertk.assertions.isNull
+import assertk.assertions.isTrue
+import assertk.assertions.prop
+import dev.stalla.builder.PodcastindexPersonBuilder
+import dev.stalla.model.anEpisodePodcastindexPerson
+import dev.stalla.model.podcastindex.PodcastindexPerson
+import org.junit.jupiter.api.Test
+
+internal class ValidatingPodcastindexPersonBuilderTest {
+
+    @Test
+    internal fun `should not build an Podcastindex Person with when all the fields are missing`() {
```

Comment:
```suggestion
    internal fun `should not build a Podcastindex Person when all the fields are missing`() {
```

Also technically redundant test